extensions/aria_extension_tosca/simple_v1_0/modeling/parameters.py

```diff
@@ -71,6 +72,7 @@ def get_assigned_and_defined_property_values(context, presentation, field_name='
     values = OrderedDict()
 
     the_type = presentation._get_type(context)
+    field_name_plural = pluralize(field_name)
```

Comment:
Previously we sent the plural as a function argument, but I optimized by removing it... just some code cleanup.